rero_ils/modules/entities/local_entities/jsonschemas/local_entities/local_entity_work-v0.0.1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "type": "object",
+  "title": "Work",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "type",
+    "authorized_access_point",
+    "title"
+  ],
```

Comment:
On principle, I agree. However, to start, the work access point will not create an entity, but will remain as it is in the document record (which preserves the subfields). Only subjects "work" will be included in the entities. For the moment, these are minimally mapped (with creator / title). When the works are included in the MEF, the work entity will be completed. In the meantime, the aim is to avoid integrating subfields that will have to be modified later...
This may not be the best way of doing things, but we won't lose any more information than we do at the moment (while importing records).